src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -95,231 +46,36 @@ const NavBar: React.FC<NavBarPropsType> = ({
-
-  const now = firebase.firestore.Timestamp.fromDate(new Date());
-  const futureUpcoming =
-    venue?.events?.filter((e) => e.ts_utc.valueOf() > now.valueOf()) ?? []; //@debt typing does this exist?
-
-  const hasUpcomingEvents = futureUpcoming && futureUpcoming.length > 0;
-
-  const hasRadioStations = radioStations && radioStations.length;
-  const isSoundCloud =
-    !!hasRadioStations && RegExp("soundcloud").test(radioStations![0]);
-
-  const sound = useMemo(
-    () =>
-      radioStations && hasElements(radioStations) && !isSoundCloud
-        ? new Audio(radioStations[0])
-        : undefined,
-    [isSoundCloud, radioStations]
-  );
-
-  const [isRadioPlaying, setIsRadioPlaying] = useState(false);
-  const { volume, setVolume } = useRadio(isRadioPlaying, sound);
-
-  const radioFirstPlayStateLoaded = useRef(false);
-  const showRadioOverlay = useMemo(() => {
-    if (!radioFirstPlayStateLoaded.current) {
-      const radioFirstPlayStorageKey = "radioFirstPlay";
-      const radioFirstPlayState = localStorage.getItem(
-        radioFirstPlayStorageKey
-      );
-      if (!radioFirstPlayState) {
-        localStorage.setItem(radioFirstPlayStorageKey, JSON.stringify(true));
-        return true;
-      }
-      radioFirstPlayStateLoaded.current = true;
-    }
-    return false;
-  }, [radioFirstPlayStateLoaded]);
-
-  const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
-  const toggleEventSchedule = useCallback(() => {
-    setEventScheduleVisible(!isEventScheduleVisible);
-  }, [isEventScheduleVisible]);
-  const hideEventSchedule = useCallback(() => {
-    setEventScheduleVisible(false);
-  }, []);
-
-  const parentVenueId = venue?.parentId ?? "";
-  const backToParentVenue = useCallback(() => {
-    window.location.href = venueInsideUrl(parentVenueId);
-  }, [parentVenueId]);
-
-  const navigateToHomepage = useCallback(() => {
-    const venueLink =
-      redirectionUrl ?? venueId ? venueInsideUrl(venueId ?? "") : "/";
-
-    window.location.href = venueLink;
-  }, [redirectionUrl, venueId]);
-
-  const handleRadioEnable = useCallback(() => setIsRadioPlaying(true), []);
-
-  const [showRadioPopover, setShowRadioPopover] = useState(false);
-
-  const toggleShowRadioPopover = useCallback(
-    () => setShowRadioPopover((prevState) => !prevState),
-    []
-  );
-
-  if (!venueId || !venue) return null;
-
-  // TODO: ideally this would find the top most parent of parents and use those details
-  const navbarTitle = parentVenue?.name ?? venue.name;
-
-  const profileImage = profile?.pictureUrl || DEFAULT_PROFILE_IMAGE;
-
-  const radioStation = !!hasRadioStations && radioStations![0];
-
-  const showNormalRadio = (venue?.showRadio && !isSoundCloud) ?? false;
-  const showSoundCloudRadio = (venue?.showRadio && isSoundCloud) ?? false;
-
   return (
     <>
       <header>
-        <div className={`navbar navbar_playa ${!isOnPlaya && "nonplaya"}`}>
+        <div className={`navbar navbar_playa nonplaya`}>
```

Comment:
```suggestion
        <div className='navbar navbar_playa nonplaya'>
```

Reply:
Presumably we can minimize this down to just a single class name that follows our BEM patterns? (eg. `NavBar`)

Reply:
These styles are in playa.scss and I don't feel comfortable change those because they are acting as global styles and I might break something. This might be a safe thing to do in the refactoring PR where playa content is being removed from the project.